index.html

```diff
@@ -29,187 +29,215 @@ <h1 class="portfolio">Portfolio</h1>
+            
+            <div class="about-skill">
 
-            <div class="skill">
-                <h3>skills</h3> 
-                <i class="fa-solid fa-angle-right"></i>
-            </div>
+                <div class="skill">
+                    <h3>Languages</h3> 
+                    <i class="fa-solid fa-angle-down"></i>
+                </div>
 
-        </div>
+                <div class="skill-content">
+                    <ul class="skill-list">
+                        <li class="skill-item">
+                            <i class="html fa-brands fa-html5 fa-2xl"></i>
+                            <p>HTML</p>
+                        </li>
+                        <li class="skill-item">
+                            <i class="css fa-brands fa-css3-alt fa-2xl"></i>
+                            <p>CSS</p>
+                        </li>
+                        <li class="skill-item">
+                            <i class="js fa-brands fa-js-square fa-2xl"></i>
+                            <p>Javascript</p>
+                        </li>
+                    </ul>
+                </div>
+                <div class="skill">
+                    <h3>Framework</h3> 
+                    <i class="fa-solid fa-angle-right"></i>
+                </div>
 
+                <div class="skill">
+                    <h3>skills</h3> 
+                    <i class="fa-solid fa-angle-right"></i>
+                </div>
+
+            </div>
+        </div>
         <form
         action="https://formspree.io/f/mwkydlpq"
         method="POST"
         class="form"
         >
-        <div class="form-des">
-            <h2>Conatct Me</h2>
-            <h3>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </h3>    
-        </div>
-
-        <input type="text" name="name" required maxlength="30" placeholder="Name...">
-
-        <input type="email" name="email" required placeholder="Email...">
+            <div class="form-des">
+                <h2>Conatct Me</h2>
+                <h3>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </h3>    
+            </div>
 
-        <textarea class="form-text" name="message" maxlength="500" required placeholder="write your message here..."></textarea>
+            <div class="form-input">
+                <input class="f-input" type="text" name="name" required maxlength="30" placeholder="Name...">
 
-        <button class="form-button" type="submit">Get in Touch</button>
+                <input class="f-input" type="email" name="email" required placeholder="Email...">
+        
+                <textarea class="f-input form-text" name="message" maxlength="500" required placeholder="write your message here..."></textarea>
+        
+                <button class="form-button" type="submit">Get in Touch</button>
```

Comment:
- [ ] Please as required your design must stick to the design as much as possible :upside_down_face: :upside_down_face:, please center your `Get in Touch` button.
See the difference bellow :point_down: :point_down: 

Your design

![Screenshot from 2022-07-08 19-31-32](https://user-images.githubusercontent.com/74501523/178042653-84325421-8166-4217-aca1-c014543a9b6e.png)

Figma design

![Screenshot from 2022-07-08 19-29-32](https://user-images.githubusercontent.com/74501523/178042701-39c4adf6-6829-4d6d-81a0-587463ac8c45.png)

Reply:
thanks for the feedback i will fix this ASAP